debug: False

# Device address/baud
dev: "/dev/ttyACM0"
baud: 115200


nAttempts: 10

loop_rate: 20

# serial timeout
sertout: 0.2

# actual marker side length in milimeters (not currently used)
markerlen: 142

